packages/arb-token-bridge-ui/src/components/common/Footer.tsx

```diff
@@ -5,34 +5,44 @@ import { DOCS_DOMAIN } from '../../constants'
 type FooterLink = {
   title: string
   href: string
-  lgOrder: number
+  className: string
```

Comment:
changed to className because tailwind JIT compilation doesn't compile variable classNames